tests/api/test_siblings.py

```diff
@@ -6,15 +6,15 @@
 
 import pytest
 
-from estuary.models.koji import KojiBuild, ContainerKojiBuild
+from estuary.models.koji import KojiBuild, ContainerKojiBuild, ModuleKojiBuild
 from estuary.models.bugzilla import BugzillaBug
 from estuary.models.distgit import DistGitCommit
 from estuary.models.errata import Advisory, ContainerAdvisory
 from estuary.models.freshmaker import FreshmakerEvent
 
 
-@pytest.mark.parametrize('resource,uid,backward_rel,expected', [
-    ('advisory', '27825', True, {
+@pytest.mark.parametrize('resource,uid,backward_rel,story_path,expected', [
```

Comment:
Instead of adding `story_path` as a parametrize parameter here, you could maybe add a second parametrize decorator that just deals with `story_path`. That way all existing scenarios are also tested with the `story_path` as module. If this is too difficult to read, you could break out the module `story_path` testing to a separate test function.